mk/VRBRAIN/Tools/README.md

```diff
@@ -0,0 +1,5 @@
+These tools are from the ros project. You can find the master sources
+here:
+
+http://github.com/ros/gencpp
+http://github.com/ros/genmsg

```

Comment:
I didn't notice that was there. It doesn't seem to make sense to have it in ArduPilot and PX4Firmware but I'm not sure if it is still needed or not. Even if it is needed to be in ArduPilot you can use the one already in the PX4 folder, no need to have this duplicated in ArduPilot (we would end up with 3 versions).